rootfs/usr/local/sbin/shell-monitor

```diff
@@ -31,25 +36,25 @@ shell_has_exited() {
 # Function to kill all active shell sessions launched by wrapper.
 # This is the shutdown procedure, so we do not care about hogging the CPU.
 kill_shells() {
-	for pid in $(list_wrapper_shells); do
+	for pid in $(list-wrapper-shells); do
```

Comment:
why are we using `-` here instead of `_`?

Reply:
Looks like its another file's function [here](https://github.com/cloudposse/geodesic/blob/main/rootfs/usr/local/sbin/list-wrapper-shells). Personal preference would be update it.

Reply:
Because we can, and by convention we use dashes for command names, which includes functions intended to be called as commands.